src/main/java/de/zalando/aruha/nakadi/service/subscription/KafkaClient.java

```diff
@@ -0,0 +1,38 @@
+package de.zalando.aruha.nakadi.service.subscription;
+
+import de.zalando.aruha.nakadi.domain.Cursor;
+import de.zalando.aruha.nakadi.domain.Subscription;
+import de.zalando.aruha.nakadi.exceptions.NakadiException;
+import de.zalando.aruha.nakadi.repository.kafka.KafkaTopicRepository;
+import de.zalando.aruha.nakadi.service.subscription.model.Partition;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Consumer;
+
+public class KafkaClient {
```

Comment:
I'm not sure that KafkaClient is a good name for this class, it's a bit confusing.